packages/calypso-ui/src/web-preview/content.jsx

```diff
@@ -47,37 +48,39 @@ export class WebPreviewContent extends Component {
 	}
 
 	componentDidMount() {
+		const { onDeviceUpdate, previewMarkup, previewUrl } = this.props;
+
 		window.addEventListener( 'message', this.handleMessage );
-		if ( this.props.previewUrl ) {
-			this.setIframeUrl( this.props.previewUrl );
+		if ( previewUrl ) {
+			this.setIframeUrl( 'https://themes.woocommerce.com/?name=galleria' );
```

Comment:
This should be `previewUrl`.

Reply:
I guess we should not force every preview here to a WC theme 🤔  - updated!